source-mysql/discovery.go

```diff
@@ -165,6 +168,12 @@ func translateRecordField(columnType interface{}, val interface{}) (interface{},
 				return val, nil
 			case "json":
 				return json.RawMessage(val), nil
+			case "datetime", "timestamp":
```

Comment:
Does this still work with existing captures & collections? I guess it would, since its still returning a string, just a more specific kind of string. Assuming that is true, I'm not entirely sure how to feel about versioning of the connector. But ultimately I think there are so few of these deployed yet that it should be fine.